winlogbeat/beat/winlogbeat.go

```diff
@@ -94,7 +95,8 @@ func (eb *Winlogbeat) Setup(b *beat.Beat) error {
 	eb.done = make(chan struct{})
 
 	var err error
-	eb.checkpoint, err = checkpoint.NewCheckpoint(eb.config.Winlogbeat.RegistryFile, 10, 5*time.Second)
+	eb.checkpoint, err = checkpoint.NewCheckpoint(
+		eb.config.Winlogbeat.RegistryFile, 10, 5*time.Second)
```

Comment:
Should this time interval be configurable or at least a constant? Or this is something recommend from windows?